src/content/tools/dart-build.md

```diff
@@ -0,0 +1,106 @@
+---
+title: dart build
+description: Command-line tool for building Dart applications.
+---
+:::version-note
+Support for `dart build` was introduced in Dart 3.10.
+:::
+
+Use the `dart build` command to build a Dart application.
+This command automatically runs [build hooks](/tools/hooks) 
+from your project and its dependencies to compile or download 
+code assets and bundle them with your application.
+
+
+## Build a CLI application 
+
+The `dart build cli` command
+builds a Dart application with a command line interface (CLI)
+including any code assets.
+
+The resulting app bundle is structured in the following manner:
+
+```
```

Comment:
We're working on a special component for showcasing file tree structures, but until then you can use `plaintext` as the language for these type of code blocks:

````suggestion
```plaintext
````